### PR TITLE
Docker support

### PR DESCRIPTION
**Pull request by daquinoaldo**
- Fix bug that prevents the bot to be run
- Switch to module (required for Docker)
- Parse telegram token from env, then from the file (required for Docker)
- Add Docker support

### CODE REVIEW

**Comment by daquinoaldo** [comment]
Sorry, wrong repo

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM python:alpine
```
Comment (macisamuele) [inline Dockerfile:1]: Please always make sure to use a well defined base image.
`python:alpine` is currently based on python3.7, what is going to happen in the future?
If you check https://github.com/macisamuele/gdgpisausermanager/blob/29e2ce660c30f0e4c94fcf2dfc2665ade6e903b0/Dockerfile you can see that I was defining a specific python and alpine version.
Build reproducibility is important (at least to me 😉 )

Comment (daquinoaldo) [reply Dockerfile:1]: Thank you for the suggestion!
Do you think that this can fix?
```suggestion
FROM python:3.7-alpine
```

Comment (macisamuele) [reply Dockerfile:1]: It could work as well (`python:3.7.3-alpine3.8` would be even better)

Comment (daquinoaldo) [reply Dockerfile:1]: Perfect! Thanks. I'll upgrade to alpine 3.9 since is the latest and works correctly on that.

**Pull request merged**